components/Checkbox.js

```diff
@@ -41,6 +41,7 @@ const Checkbox = ({ className, id, labelText, onChange, ...other }) => {
           <Icon
             className="bx--checkbox-checkmark"
             description="checkmark"
+            alt="checkmark"
```

Comment:
It was pointed out in https://github.com/carbon-design-system/carbon-components-react/issues/87#issuecomment-314853593 that we need to add `alt` attributes to all of our `Icon` components any way. 

So in `Icon`, couldn't we just add `alt` as a prop and as a defaultProp?